### PR TITLE
Ben - LA100 CDIAC downscale ctry

### PR DESCRIPTION
**Pull request by bpbond**
Who else should be assigned as a reviewer here, who works with/knows CDIAC data?

### CODE REVIEW

File: R/zchunk_LA100.CDIAC_downscale_ctry.R [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+      mutate(share = value / sum(value)) %>%
+      select(iso, share, category) %>%
+      # Use those share values to project USSR totals back in time
+      right_join(L100.CO2_USSR_hist_repCtry, by = c("iso", "category")) %>%
+      mutate(value = value * share) %>%
+      select(-share) %>%
+      spread(category, value) ->
+      L100.CO2_FSU_hist
+
+    # Repeat: generate time series of Yugoslavia by country, using national shares in the first available year
```
Comment (mwisepnnl) [inline R/zchunk_LA100.CDIAC_downscale_ctry.R]: explain why this is necessary

Comment (bpbond) [inline R/zchunk_LA100.CDIAC_downscale_ctry.R]: Done.

File: R/zchunk_LA100.CDIAC_downscale_ctry.R [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+             FILE = "energy/CDIAC_Cseq_by_nation",
+             FILE = "energy/CDIAC_nation"))
+  } else if(command == driver.DECLARE_OUTPUTS) {
+    return(c("L100.CDIAC_CO2_ctry_hist"))
+  } else if(command == driver.MAKE) {
+
+    all_data <- list(...)[[1]]
+
+    # Load required inputs
+    CDIAC_CO2_by_nation <- get_data(all_data, "energy/CDIAC_CO2_by_nation")
```
Comment (mwisepnnl) [inline R/zchunk_LA100.CDIAC_downscale_ctry.R]: @bpbond @pkyle  why are these files being pulled from energy and not emissions (like they were in the old data system)

Comment (bpbond) [inline R/zchunk_LA100.CDIAC_downscale_ctry.R]: That looks like a mistake by the script the set up the new files; the original data system lines are
```R
CDIAC_CO2_by_nation <- readdata( "EMISSIONS_LEVEL0_DATA", "CDIAC_CO2_by_nation" )
CDIAC_Cseq_by_nation <- readdata( "EMISSIONS_LEVEL0_DATA", "CDIAC_Cseq_by_nation" )
CDIAC_nation <- readdata( "EMISSIONS_MAPPINGS", "CDIAC_nation" )
```

I'll move the files and update this PR.

**Pull request merged**